reconcile/openshift_clusterrolebindings.py

```diff
@@ -45,119 +55,162 @@
 
 QONTRACT_INTEGRATION = "openshift-clusterrolebindings"
 QONTRACT_INTEGRATION_VERSION = make_semver(0, 1, 0)
+NAMESPACE_CLUSTER_SCOPE = "cluster"
+
+
+class OCResource(BaseModel, arbitrary_types_allowed=True):
+    resource: OR
+    resource_name: str
+
+
+@dataclass
```

Comment:
Could you please not mix dataclasses and Pydantic models? Just use a basemodel here too; you don't need to set `arbitrary_types_allowed=True` below.